pycbc/inference/sampler/multinest.py

```diff
@@ -73,6 +75,11 @@ def __init__(self, model, nlivepoints, checkpoint_interval=1000,
             logging.getLogger().setLevel(loglevel)
         except ImportError:
             raise ImportError("pymultinest is not installed.")
+        try:
```

Comment:
Second part duplicates what is already in https://github.com/gwastro/pycbc/blob/master/pycbc/pool.py. Can you verify and if so consolidate?

Reply:
Thanks for pointing that out @ahnitz, I've changed it reference the existing code.